src/database/queries/applicationForms/insertOne.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO application_forms (
+  opportunity_id,
+  version
+) VALUES (
+  :opportunityId,
+  COALESCE(
```

Comment:
I wondered if coalesce was needed here due to the default value, but I guess the situation is "this might be the first form for the given opportunity" and therefore the subquery returns null, so it makes sense. I guess this makes the versions autoincrementing within a given opportunity and that's convenient.